clang/lib/CodeGen/CGDecl.cpp

```diff
@@ -362,6 +362,8 @@ CodeGenFunction::AddInitializerToStaticVarDecl(const VarDecl &D,
     return GV;
   }
 
+  PGO.markStmtMaybeUsed(D.getInit()); // FIXME: Too lazy
```

Comment:
Too lazy?

Reply:
It marks entire the initializer as "don't check further". Just for checks and will not affect actual code generation.

I'll sink it to other places when we will get issues.

Reply:
OK gotcha